Frameworks/UIKit/UIDatePicker.mm

```diff
@@ -843,6 +857,8 @@ - (void)dealloc {
     _dayStartDate = nil;
     _pickerView = nil;
     _months = nil;
+    _minimumDate = nil;
```

Comment:
Since the `AutoId` destructor does the release for you, it's not necessary to stomp it with nil in dealloc! :+1: on this change though!